--- /opt/backup/etc/lighttpd/conf.d/proxy.conf
+++ /etc/lighttpd/conf.d/proxy.conf
@@ -1,5 +1,5 @@
 $HTTP["url"] !~ "^/(config/)|(upnp/)|(webui/)|(ise/)|(api/)|(tools/)|(pda)|(pages/jpages)|(addons).*" {
-  proxy.server = ("/" => (( "host" => "127.0.0.1", "port" => 8183 )))
+  proxy.server = ("/" => (( "host" => "127.0.0.1", "port" => 8181 )))
 }
 
 $HTTP["url"] =~ "^/(pages/jpages).*" {
--- /opt/backup/etc/config/rega.conf
+++ /etc/rega.conf
@@ -42,7 +42,7 @@
 # HttpListenerPort
 # Portnummer des WebServers
 # Standardwert: 8181
-HttpListenerPort=8183
+HttpListenerPort=8181
 
 # VirtualPathInternalConfig
 # Standard: /config/
--- /opt/backup/etc/rega_http.port
+++ /etc/rega_http.port
@@ -1 +1 @@
-8183
+8181
