diff -ru --no-dereference -x '*.lock' -x lib -x firmware -x home -x media -x mnt -x opt -x proc -x root -x run -x sbin -x srv -x sys -x tmp -x lib32 -x lib64 -x lost+found -x usr -x bin -x boot -x dev -x log -x .java /opt/backup/etc/config/rfd.conf /etc/config/rfd.conf
--- /opt/backup/etc/config/rfd.conf	2018-12-19 18:48:57.843182385 +0100
+++ /etc/config/rfd.conf	2018-12-19 18:49:44.664007545 +0100
@@ -16,12 +16,12 @@
 Address File = /etc/config/ids
 Firmware Dir = /firmware
 Replacemap File = /firmware/rftypes/replaceMap/rfReplaceMap.xml
-Fire NACK Error Events = true
-Improved Coprocessor Initialization = true
+#Fire NACK Error Events = true
+#Improved Coprocessor Initialization = true
 
-[Interface 0]
-Type = CCU2
-ComPortFile = /dev/mmd_bidcos
+#[Interface 0]
+#Type = CCU2
+#ComPortFile = /dev/mmd_bidcos
 #AccessFile = /dev/null
 #ResetFile = /dev/ccu2-ic200
 
