--- /opt/backup/etc/lighttpd/conf.d/cgi.conf
+++ /etc/lighttpd/conf.d/cgi.conf
@@ -1,5 +1,5 @@
 #### CGI module
 cgi.assign = ( ".pl"  => "/usr/bin/perl",
-                ".cgi" => "/opt/hm/bin/tclsh",
+                ".cgi" => "/bin/tclsh",
                 ".ccc" => ""
              )
--- /opt/backup/etc/lighttpd/lighttpd.conf
+++ /etc/lighttpd/lighttpd.conf
@@ -14,10 +14,10 @@
 ## chroot example aswell.
 ##
 var.log_root    = "/var/log"
-var.server_root = "/opt/hm/www"
+var.server_root = "/www"
 var.state_dir   = "/var/run"
 var.home_dir    = "/var/lib/lighttpd"
-var.conf_dir    = "/opt/hm/etc/lighttpd"
+var.conf_dir    = "/etc/lighttpd"
 
 ## 
 ## run the server chrooted.

