--- occu/firmware/HM-MOD-UART/fwmap.orig
+++ /firmware/HM-MOD-UART/fwmap
@@ -1,7 +1,7 @@
 #Type           Filename                Version
 
 #Coprozessor
-CCU2                    coprocessor_update.eq3                         	1.4.1		# CoProzessor HM-MOD-UART
+#CCU2                    coprocessor_update.eq3                         	1.4.1		# CoProzessor HM-MOD-UART
 
-#CCU2			dualcopro_si1002_update_blhm.eq3		2.8.6		# Dual CoProzessor HM-MOD-UART
+CCU2			dualcopro_si1002_update_blhm.eq3		2.8.6		# Dual CoProzessor HM-MOD-UART
 
--- occu/WebUI/www/rega/index.htm.orig
+++ /www/rega/index.htm
@@ -89,7 +89,11 @@
     
       if (true == checkBrowser())
       {
-        sStartPage = STARTPAGE_PC + "?sid=" + SessionId;
+        if(typeof(SessionId) === 'undefined') {
+          sStartPage = STARTPAGE_PC;
+        } else {
+          sStartPage = STARTPAGE_PC + "?sid=" + SessionId;
+        }
         <%
           if ( (false == system.IsVar("NoAutoLogin")) || (false == system.GetVar("NoAutoLogin")) )
           {
