--- /opt/backup/www/config/cp_maintenance.cgi
+++ /www/config/cp_maintenance.cgi
@@ -1125,7 +1125,7 @@
 
 proc action_download_logfile {} {
   set HOSTNAME [exec hostname]
-  set iso8601_date [exec date -Iseconds]
+  set iso8601_date [exec date +%Y-%m-%dT%H:%M:%S%z]
   regexp {^(\d+)-(\d+)-(\d+)T(\d+):(\d+):(\d+)([+-]\d+)$} $iso8601_date dummy year month day hour minute second zone
   
   puts "Content-Type:application/x-download"
--- /opt/backup/www/config/cp_security.cgi
+++ /www/config/cp_security.cgi
@@ -1465,7 +1465,7 @@
 
 proc action_create_backup {} {
   set HOSTNAME [exec hostname]
-  set iso8601_date [exec date -Iseconds]
+  set iso8601_date [exec date +%Y-%m-%dT%H:%M:%S%z]
   regexp {^(\d+)-(\d+)-(\d+)T(\d+):(\d+):(\d+)([+-]\d+)$} $iso8601_date dummy year month day hour minute second zone
   #save DOM
   rega system.Save()
--- /opt/backup/www/config/cp_time.cgi
+++ /www/config/cp_time.cgi
@@ -741,7 +741,7 @@
 proc action_put_page {} {
   global env LOCATIONS TIMEZONES
   
-  set iso8601_date [exec date -Iseconds]
+  set iso8601_date [exec date +%Y-%m-%dT%H:%M:%S%z]
   regexp {^(\d+)-(\d+)-(\d+)T(\d+):(\d+):(\d+)([+-]\d+)$} $iso8601_date dummy year month day hour minute second zone
   
   puts "<div id=\"dragTime\">"
