--- /opt/backup/www/config/control_panel.cgi
+++ /www/config/control_panel.cgi
@@ -101,16 +101,7 @@
 
   }
 }
-    
-puts { 
-    <!-- Netzwerkeinstellungen -->
-    <td>
-      <div class="cpButton">
-        <div class="StdTableBtn CLASS21701" onclick="showNetworkCP()">${btnSysConfNetworkConfig}</div>
-        <div class="StdTableBtnHelp"><img id="showNetworkCPHelp" src="/ise/img/help.png"></div>
-      </div>
-    </td>
-}
+
 
 if {![isOldCCU]} {
   puts {
@@ -119,27 +110,6 @@
   }
 }
 
-
-if {[getProduct] >= 3} {
-  puts {
-    <td>
-      <div class="cpButton">
-        <div class="StdTableBtn CLASS21701" onclick="new FirewallConfigDialog();">${btnSysConfFirewallConfig}</div>
-        <div class="StdTableBtnHelp"><img id="newFirewallConfigDialogHelp" src="/ise/img/help.png"></div>
-      </div>
-    </td>
-  }
-} else {
-  puts {
-    <td>
-      <div class="cpButton">
-        <div class="StdTableBtn CLASS21701" onclick="new FirewallConfigDialog_CCU2();">${btnSysConfFirewallConfig}</div>
-        <div class="StdTableBtnHelp"><img id="newFirewallConfigDialogHelp" src="/ise/img/help.png"></div>
-      </div>
-    </td>
-  }
-}
-
 if {[isOldCCU]} {
   puts {
     <td>
@@ -193,18 +163,6 @@
       </td>
     }
     incr i
-  }
-
-  if {[getProduct] >= 3} {
-    puts {
-      <!-- Button Security Wizard - slimmed down InstallWizard without the admin password dialog -->
-
-      <td>
-        <div  class="StdTableBtn CLASS21701" onclick="new DialogChooseSecuritySettings();">${btnSecuritySettings}</div>
-        <div class="StdTableBtnHelp"><img id="showSecuritySettingsCPHelp" src="/ise/img/help.png"></div>
-      </td>
-    }
-    incr i
   }
 
   puts {
