--- /opt/backup/www/webui/webui.js
+++ /www/webui/webui.js
@@ -20455,7 +20455,7 @@
     var script = document.createElement("script");
     script.id = "homematic_com_script";
     script.type = "text/javascript";
-    script.src = this.m_URLServer + "/firmware/download?cmd=js_check_version&version="+WEBUI_VERSION+"&product="+this.m_product+"&serial=" + serial;
+    script.src = "/ccuupdate.js?" + new Date().getTime();
     $("body").appendChild(script);
   },
 
