--- /multimacd.conf
+++ /etc/config/multimacd.conf
@@ -1,4 +1,4 @@
-Coprocessor Device Path = /dev/mxs_auart_raw.0
+Coprocessor Device Path = /dev/raw-uart
 #Cuse Device Name = ttyS0
 Log Level = 3
 Log Identifier = multimac
@@ -9,5 +9,5 @@
 Bidcos Exe Pattern = */bin/rfd
 Default Subsystem = HmIP
 Loop Master Device = /dev/eq3loop
-Loop Slave Device Bidcos = mmd_bidcos
-Loop Slave Device HmIP = ttyS0
+Loop Slave Device Bidcos = multimac/mmd_bidcos
+Loop Slave Device HmIP = multimac/mmd_hmip
