--- /opt/occu/X86_32_Debian_Wheezy/packages-eQ-3/RFD/etc/config_templates/multimacd.conf
+++ /etc/config/multimacd.conf
@@ -1,5 +1,5 @@
-Coprocessor Device Path = /dev/mxs_auart_raw.0
-#Cuse Device Name = ttyS0
+Coprocessor Device Path = /dev/raw-uart
+Cuse Device Name = mmd_hmip
 Log Level = 3
 Log Identifier = multimac
 Log Destination = Syslog
@@ -10,4 +10,4 @@
 Default Subsystem = HmIP
 Loop Master Device = /dev/eq3loop
 Loop Slave Device Bidcos = mmd_bidcos
-Loop Slave Device HmIP = ttyS0
+Loop Slave Device HmIP = mmd_hmip
