--- /opt/backup/www/config/cp_maintenance.cgi
+++ /www/config/cp_maintenance.cgi
@@ -361,126 +361,126 @@
     #set styleMaxWidth "style=max-width:70px;"
 
     table {class="popupTable CLASS20901 j_translate"} {border="1"} {
-      table_row {class="CLASS20902"} {
-        table_data {class="CLASS20903"} $styleMaxWidth {
+#      table_row {class="CLASS20902"} {
+#        table_data {class="CLASS20903"} $styleMaxWidth {
           #puts "Zentralen-<br>"
           #puts "Software"
-          puts "\${dialogSettingsCMTDCCUSoftware}"
-        }
-        table_data {class="CLASS20904"} {width="400"} {
-          table {class="CLASS20909"} {
-            set cur_version [get_version]
-            set serial [get_serial]
-            table_row {
-              table_data {align="left"} {colspan="2" id="actualSWVersion"} {
-                puts "\${dialogSettingsCMLblActualSoftwareVersion}"
-              }
-              table_data {
-                puts "$cur_version"
-              }
-            }
-            table_row {
-              table_data {align="left"} {colspan="2"} {
-                puts "\${dialogSettingsCMLblAvailableSoftwareVersion}"
-              }
-              table_data {id="availableSWVersion"} {
+#          puts "\${dialogSettingsCMTDCCUSoftware}"
+#        }
+#        table_data {class="CLASS20904"} {width="400"} {
+#          table {class="CLASS20909"} {
+#            set cur_version [get_version]
+#            set serial [get_serial]
+#            table_row {
+#              table_data {align="left"} {colspan="2" id="actualSWVersion"} {
+#                puts "\${dialogSettingsCMLblActualSoftwareVersion}"
+#              }
+#              table_data {
+#                puts "$cur_version"
+#              }
+#            }
+#            table_row {
+#              table_data {align="left"} {colspan="2"} {
+#                puts "\${dialogSettingsCMLblAvailableSoftwareVersion}"
+#              }
+#              table_data {id="availableSWVersion"} {
                 # This doesn´t work properly
                 # puts [iframe "$REMOTE_FIRMWARE_SCRIPT?cmd=check_version&version=$cur_version&serial=$serial&lang=de&product=HM-CCU3" marginheight=0 marginwidth=0 frameborder=0 width=100 height=20 {scrolling="no"} ]
                 # The available version will be set further down with "jQuery('#availableSWVersion').html(homematic.com.getLatestVersion());"
-              }
-            }
-            table_row {
-              table_data {align="left"} {colspan="3"} {
+#              }
+#            }
+#            table_row {
+#              table_data {align="left"} {colspan="3"} {
                 #puts "[bold "Software-Update durchführen"]"
-                puts "<b>\${dialogSettingsCMLblPerformSoftwareUpdate}</b>"
-              }
-            }
-            table_row {
-              td {width="20"} {}
-              table_data {align="left"} {
-                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep1}"
-              }
-              table_data {
-                division {class="popupControls CLASS20905"} {
-                  table {
-                    table_row {
-                      table_data {
-                        division {class="CLASS20908" style="display: none"} {id="btnFwDownload"} {} "onClick=\"window.location.href='$REMOTE_FIRMWARE_SCRIPT?cmd=download&version=$cur_version&serial=$serial&lang=de&product=HM-CCU3';\"" {}
-                        division {class="CLASS20908"}  "onClick=\"showCCULicense();\"" {puts "\${dialogSettingsCMBtnPerformSoftwareUpdateDownload}"}
-                      }
-                    }
-                  }
-                }
-              }
-            }
-            table_row {
-              td {width="20"} {}
-              table_data {align="left"} {colspan="2"} {
-                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep2}"
-              }
-            }
-            table_row {
-              td {width="20"} {}
-              table_data {colspan="2"} {
-                form "$env(SCRIPT_NAME)?sid=$sid" name=firmware_form {target=firmware_upload_iframe} enctype=multipart/form-data method=post {
-                  export action=firmware_upload
-                  export downloadOnly=$downloadOnly
-                  file_button firmware_file size=30 maxlength=1000000
-                }
-                puts {<iframe name="firmware_upload_iframe" style="display: none;"></iframe>}
-              }
-            }
-            table_row {
-              td {width="20"} {}
-              table_data {align="left"} {
-                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep3}"
-              }
-              table_data {
-                division {class="popupControls CLASS20905"} {
-                  table {
-                    table_row {
-                      table_data {
-                        division {class="CLASS20919"} {onClick="document.firmware_form.submit();showUserHint();"} {
-                          puts "\${dialogSettingsCMBtnPerformSoftwareUpdateUpload}"
-                        }
-                      }
-                    }
-                  }
-                }
-              }
-            }
-            table_row {
-              td {width="20"} {}
-              table_data {align="left"} {colspan="2"} {class="CLASS20920"} {
-                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep4}"
-              }
-            }
-          }
-        }
-        table_data {align="left"} {class="CLASS20921"} {
-          puts "\${dialogSettingsCMHintSoftwareUpdate1}"
-          number_list {class="j_noForcedUpdate"} {
-            li {
-              ${dialogSettingsCMHintSoftwareUpdate2}            }
-            li {
-               ${dialogSettingsCMHintSoftwareUpdate3}
-            }
-            li {
-               ${dialogSettingsCMHintSoftwareUpdate3a}
-            }
-            set bat_level [get_bat_level]
-            if {$bat_level < 50} {
-              set msg " \${dialogSettingsCMHintSoftwareUpdate4a} $bat_level%. "
-              append msg  \${dialogSettingsCMHintSoftwareUpdate4b}
-              li $msg
-            }
-          }
-
-          division {class="j_forcedUpdate" style="padding:10px;"} {
-            puts "<br/>\${dialogSettingsCMHintSoftwareUpdate2}"
-          }
-        }
-      }
+#                puts "<b>\${dialogSettingsCMLblPerformSoftwareUpdate}</b>"
+#              }
+#            }
+#            table_row {
+#              td {width="20"} {}
+#              table_data {align="left"} {
+#                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep1}"
+#              }
+#              table_data {
+#                division {class="popupControls CLASS20905"} {
+#                  table {
+#                    table_row {
+#                      table_data {
+#                        division {class="CLASS20908" style="display: none"} {id="btnFwDownload"} {} "onClick=\"window.location.href='$REMOTE_FIRMWARE_SCRIPT?cmd=download&version=$cur_version&serial=$serial&lang=de&product=HM-CCU3';\"" {}
+#                        division {class="CLASS20908"}  "onClick=\"showCCULicense();\"" {puts "\${dialogSettingsCMBtnPerformSoftwareUpdateDownload}"}
+#                      }
+#                    }
+#                  }
+#                }
+#              }
+#            }
+#            table_row {
+#              td {width="20"} {}
+#              table_data {align="left"} {colspan="2"} {
+#                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep2}"
+#              }
+#            }
+#            table_row {
+#              td {width="20"} {}
+#              table_data {colspan="2"} {
+#                form "$env(SCRIPT_NAME)?sid=$sid" name=firmware_form {target=firmware_upload_iframe} enctype=multipart/form-data method=post {
+#                  export action=firmware_upload
+#                  export downloadOnly=$downloadOnly
+#                  file_button firmware_file size=30 maxlength=1000000
+#                }
+#                puts {<iframe name="firmware_upload_iframe" style="display: none;"></iframe>}
+#              }
+#            }
+#            table_row {
+#              td {width="20"} {}
+#              table_data {align="left"} {
+#                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep3}"
+#              }
+#              table_data {
+#                division {class="popupControls CLASS20905"} {
+#                  table {
+#                    table_row {
+#                      table_data {
+#                        division {class="CLASS20919"} {onClick="document.firmware_form.submit();showUserHint();"} {
+#                          puts "\${dialogSettingsCMBtnPerformSoftwareUpdateUpload}"
+#                        }
+#                      }
+#                    }
+#                  }
+#                }
+#              }
+#            }
+#            table_row {
+#              td {width="20"} {}
+#              table_data {align="left"} {colspan="2"} {class="CLASS20920"} {
+#                puts "\${dialogSettingsCMLblPerformSoftwareUpdateStep4}"
+#              }
+#            }
+#          }
+#        }
+#        table_data {align="left"} {class="CLASS20921"} {
+#          puts "\${dialogSettingsCMHintSoftwareUpdate1}"
+#          number_list {class="j_noForcedUpdate"} {
+#            li {
+#              ${dialogSettingsCMHintSoftwareUpdate2}            }
+#            li {
+#               ${dialogSettingsCMHintSoftwareUpdate3}
+#            }
+#            li {
+#               ${dialogSettingsCMHintSoftwareUpdate3a}
+#            }
+#            set bat_level [get_bat_level]
+#            if {$bat_level < 50} {
+#              set msg " \${dialogSettingsCMHintSoftwareUpdate4a} $bat_level%. "
+#              append msg  \${dialogSettingsCMHintSoftwareUpdate4b}
+#              li $msg
+#            }
+#          }
+
+#          division {class="j_forcedUpdate" style="padding:10px;"} {
+#            puts "<br/>\${dialogSettingsCMHintSoftwareUpdate2}"
+#          }
+#        }
+#      }
       table_row {class="CLASS20902 j_noForcedUpdate j_fwUpdateOnly"} {
         table_data {class="CLASS20903"} $styleMaxWidth {
           #puts "Zentralen-<br>"
@@ -519,23 +519,23 @@
       }
 
       # Abgesicherter Modus
-      table_row {class="CLASS20902 j_noForcedUpdate j_fwUpdateOnly"} {
-        table_data {class="CLASS20903"} $styleMaxWidth {
+#      table_row {class="CLASS20902 j_noForcedUpdate j_fwUpdateOnly"} {
+#        table_data {class="CLASS20903"} $styleMaxWidth {
           #puts "Abgesicherter<br>"
           #puts "Modus"
-          puts "\${dialogSettingsCMTDCCUSafeMode}"
-        }
-        table_data {class="CLASS20904"} {
-          division {class="popupControls CLASS20905"} {
-            division {class="CLASS20910 colorGradient50px"} {onClick="OnEnterSafeMode();"} {
-              puts "\${dialogSettingsCMBtnCCURestartSafe}"
-            }
-          }
-        }
-        table_data {align="left"} {class="CLASS20904"} {
-          puts "\${dialogSettingsCMHintRestartSafeMode}"
-        }
-      }
+#          puts "\${dialogSettingsCMTDCCUSafeMode}"
+#        }
+#        table_data {class="CLASS20904"} {
+#          division {class="popupControls CLASS20905"} {
+#            division {class="CLASS20910 colorGradient50px"} {onClick="OnEnterSafeMode();"} {
+#              puts "\${dialogSettingsCMBtnCCURestartSafe}"
+#            }
+#          }
+#        }
+#        table_data {align="left"} {class="CLASS20904"} {
+#          puts "\${dialogSettingsCMHintRestartSafeMode}"
+#        }
+#      }
 
       table_row {class="CLASS20902 j_noForcedUpdate j_fwUpdateOnly"} {
         table_data {class="CLASS20903"} $styleMaxWidth  {
