--- /opt/backup/www/config/control_panel.cgi
+++ /www/config/control_panel.cgi
@@ -104,7 +104,14 @@
 #      </div>
 #    </td>
 #}
-
+puts {
+    <!-- CCU-Update -->
+    <td>
+      <div class="cpButton">
+        <div class="StdTableBtn CLASS21701" onclick="ccuUpdateCP()">CCU Update</div>
+      </div>
+    </td>
+}
 if {![isOldCCU]} {
   puts {
     </tr>
--- /opt/backup/www/webui/webui.js
+++ /www/webui/webui.js
@@ -37318,6 +37318,10 @@
  {
   CreateCPPopup("/config/cp_network.cgi");
 };
+ccuUpdateCP = function()
+ {
+  CreateCPPopup("/config/cp_ccuupdate.cgi");
+};
 showGeneralSettingsCP = function()
  {
   CreateCPPopup("/pages/jpages/system/StorageSettings/show");
